src/api/python/CMakeLists.txt

```
@@ -19,6 +19,37 @@ if(POLICY CMP0057)
   cmake_policy(SET CMP0057 NEW)
 endif()
 
+# Check that scikit-build is installed
+# Provides CMake files for Python bindings
+check_python_module("skbuild" "scikit-build")
+
+# setuptools for installing the module
+check_python_module("setuptools")
+
+# Find cmake modules distributed with scikit-build
+# They are distributed under <scikit-build directory>/resources/cmake
+execute_process(
+  COMMAND
+    ${PYTHON_EXECUTABLE} -c "from __future__ import print_function; \
+                             import os; import skbuild; \
+                             cmake_module_path=os.path.dirname(skbuild.__file__) + \
+                             os.path.sep + 'resources' + os.path.sep + 'cmake'; \
+                             assert os.path.isdir(cmake_module_path); \
+                             print(cmake_module_path, end='')"
```

```suggestion
    ${PYTHON_EXECUTABLE} -c "from __future__ import print_function; \
                             import os; import skbuild; \
                             cmake_module_path=os.path.join(os.path.dirname(skbuild.__file__),
                             'resources', 'cmake'; print(cmake_module_path, end='')"
```
I'd check the path in cmake with `if(EXISTS ...)` since you can provide a better error message

---

Good idea, addressed with 8764161ad7564fb7d80555c43c940ee3d7dd9a98